nilearn/decoding/searchlight.py

```diff
@@ -287,6 +287,9 @@ def __init__(
         self.scoring = scoring
         self.cv = cv
         self.verbose = verbose
+        self.scores_ = None
```

Comment:
Can you update the doc string by adding an Attributes section that describes what those are?